pages/price-feeds/use-real-time-data/solana.mdx

```diff
@@ -115,16 +127,21 @@ const solUsdPriceFeedAccount = pythSolanaReceiver
   .toBase58();
 ```
 
-The price feed account integration assumes that an off-chain process is continuously updating each price feed.
-The Pyth Data Association sponsors price updates for a subset of commonly-used price feeds on shard 0.
-Please see [Sponsored Feeds](/price-feeds/sponsored-feeds) for a list of sponsored feeds and their account addresses.
-However, updating a price feed is a permissionless operation, and anyone can run this process.
-Please see [Using Scheduler](/price-feeds/schedule-price-updates/using-scheduler) for more information.
-Running the scheduler can help with reliability and to update feed/shard pairs that are not part of the default schedule.
+<Callout type="info" emoji="ℹ️">
+  **NOTE:** The Price Feed Accounts integration assumes that an off-chain
```

Comment:
Whiy is Price Feed Account capitalized here. It's not capitalized above. I suggest not capitalizing.